Zone5/Transport/StringEncodedBody.swift

```diff
@@ -8,7 +8,7 @@
 
 import Foundation
 
-struct StringEncodedBody: JSONEncodedBody {
+public struct StringEncodedBody: JSONEncodedBody {
```

Comment:
Does this need to be public? Ride does not need this struct.

Reply:
yes. If you have a custom request that takes a simple string as the post body you can use this object.